app/assets/javascripts/components/articles/article.jsx

```diff
@@ -21,7 +21,7 @@ const Article = React.createClass({
 
   shouldShowLanguagePrefix() {
     if (!this.props.course.home_wiki) { return false; }
-    return this.props.article.language !== this.props.course.home_wiki.language;
+    return this.props.article.language !== this.props.course.home_wiki.language && null;
```

Comment:
I'm not clear about what the `&& null` is doing here. It looks like this would always return either `false` or `null`.

Reply:
yes, you are right. reworking on it right now, sorry.